apps/server/src/shared/domain/entity/user.entity.ts

```diff
@@ -45,6 +46,7 @@ interface UserInfo {
 @Entity({ tableName: 'users' })
 @Index({ properties: ['id', 'email'] })
 @Index({ properties: ['firstName', 'lastName'] })
+@Index({ properties: ['preferredName'] })
```

Comment:
I don't think we need an index for it yet, since we don't have any specific use cases for this attribute.